src/tatm/utils.py

```diff
@@ -1,8 +1,21 @@
 import logging
+from enum import Enum
 
 
 def configure_logging(log_level=logging.INFO):
     logging.basicConfig(
         level=log_level,
         format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
     )
+
+
+class TatmOptionEnum(str, Enum):
```

Comment:
There is a `StrEnum` class that you can use instead of `(str, Enum)`.

Reply:
Unfortunately we don't have that in python 10

Reply:
Oof, good catch. Will approve.